storybook/stand-alone-documentation/welcome/welcome.stories.mdx

```diff
@@ -0,0 +1,24 @@
+import { Meta } from "@storybook/blocks";
+import { WelcomeIntro } from "./intro/intro";
+import { Contributors } from "./contributors/contributors";
+import { WelcomeHeader } from "./welcome-header/welcome-header";
+import { OtherContributorsList } from "./other-contributors-list/other-contributors-list";
+
+<Meta title="Welcome" />
+
+<WelcomeHeader />
+<WelcomeIntro />
+
+## Resources
+
+- [Github](https://github.com/mondaycom/monday-ui-style/)
+
+## Contributors
+
+monday.com’s designers, developers, and writers provide teams with universal assets—elements, components, patterns, and code—and the guidance on how to design and build with them.
+We welcome contributions to monday.com design system!
+
+### Meet the Vibe team
+
+<Contributors />
+<OtherContributorsList />
```

Comment:
I think you can remove it for now. It's a lot of code and it's a duplication of the vibe page which means double maintenance

Reply:
I'd like to leave it as it is
- it just looks better with Welcome page
- It gives more consistent experience regarding Vibe storybook
- We can later fill it with more useful information
- Double maintenance will only be needed when will replace team pictures, the rest of the content is different
-  I've already put some efforts to do it

Reply:
it's just that there's so much duplication between this and the other storybooks so it's terrible for maintenance.